src/gui.cc

```diff
@@ -490,10 +490,11 @@ void GUI::DisplayTimeline(
         TimelineDateAt().year(),
         TimelineDateAt().month(),
         TimelineDateAt().day());
+    int tzd = datetime.tzd();
 
     // Get all entires in this day (no chunk, no overlap)
     TogglTimeEntryView *first_entry = nullptr;
-    time_t start_day = datetime.timestamp().epochTime();
+    time_t start_day = datetime.timestamp().epochTime() - tzd;
```

Comment:
We have to minus the Timezone different since the EpochTime is at GTM 0, but we need to fetch all TimeEntry from the midnight of current time zone (+7 for instance).

If we have any better solution, please comment below 👍

Reply:
No better solution comes to mind. Let's go with this one.